README.md

```diff
@@ -64,11 +64,14 @@ fastify.register(require('@fastify/postgres'), {
 
 fastify.get('/user/:id', async (req, reply) => {
   const client = await fastify.pg.connect()
-  const { rows } = await client.query(
-    'SELECT id, username, hash, salt FROM users WHERE id=$1', [req.params.id],
-  )
-  client.release()
-  return rows
+  try {
+    const { rows } = await client.query(
+      'SELECT id, username, hash, salt FROM users WHERE id=$1', [req.params.id],
+    )
+    return rows
+  } finally {
+    client.release()
```

Comment:
One thing to bear in mind here is you shouldn’t do expensive comp on the rows before releasing, release as soon as you get the query result. This can easily be achieved by wrapping this part in a fn or using let instead of const.

Reply:
(This should be fairly obvious to a reader and beyond the responsibility of this project imho)

Reply:
Mind adding that to your PR?